### PR TITLE
[sys_utils] Fixed getty host name display

### PR DESCRIPTION
**Pull request by Mellvik**
In order to display the host name as part of the login prompt, `getty` used to pick the host name from the first line of `/etc/hosts`, a relic from before TLVC/ELKS had TCP/IP and a 'real' `/etc/hosts` file.

The fix deletes the old code and uses the HOSTNAME environment variable instead. More reliable and simpler than using the 'new' `/etc/hosts`.

Also included is a compile time option to print the jiffies value at startup time, a part of a time measurement mechanism (debug) in which the kernel init (`init/main.c`) prints the starting value, `getty` the ending value.

### CODE REVIEW

**Comment by ghaerr** [comment]
You might also be interested in [this bug fix](https://github.com/ghaerr/elks/pull/2021).

Are you getting rid of the /etc/hosts functionality in total, or just using HOSTNAME= instead of reading /etc/hosts? I'm not clear on exactly what the benefit is here, as I think /etc/hosts is still required in certain libc routines for converting hosts to IP  addresses or vice versa.

**Comment by Mellvik** [comment]
Thanks for the link/fix, I'll certainly include that!

This is a bugfix and an optimization. Got rid of a big (!) static buffer and a bunch of code in addition to the fix itself. Getty made wrong (ancient) and clumsy assumptions about the hosts file. This is simpler, and /etc/hosts lives on.

File: tlvccmd/sys_utils/getty.c [hunk 1]
```diff
@@ -86,28 +90,16 @@ void consolemsg(const char *str, ...)
...
-	ptr--;
-    *ptr = '\0';
-    if (!*Host)
-	strcpy( Host, "LocalHost" );
+    char *host = getenv("HOSTNAME");
+
+    if (host == NULL)
+	strcpy(Host, "LocalHost");
+    else
+	strcpy(Host, host);
```
Comment (ghaerr) [inline tlvccmd/sys_utils/getty.c:102]: If you're trying for min executable size, replace line 100 strcpy with initialization of char Host[] with Localhost via declaration statement. 

Line 102 strcpy is dangerous as length is not checked. Recommend `strncpy(Host, host, sizeof(Host)-1)`.

Comment (Mellvik) [reply tlvccmd/sys_utils/getty.c:102]: Thanks, this is good. And yes, definitely going for min size. Along with `init`, `getty` is the one program always running. A byte here, a byte there...

**Pull request merged**